spanner/unit_tests/test_client.py

```diff
@@ -114,14 +151,17 @@ def test_instance_admin_api(self):
         client = self._make_one(project=self.PROJECT, credentials=creds)
 
         class _Client(object):
-            pass
+            def __init__(self, *args, **kwargs):
+                self.args = args
+                self.kwargs = kwargs
```

Comment:
I asked the same question. I _think_ the reason @tseaver did what he did was for the instance check. I am doing path of least resistance.